tests/unit/style/parsing/effects.rs

```diff
@@ -33,3 +34,23 @@ fn test_clip() {
                                    "rect(auto, auto, auto, auto)");
 }
 
+#[test]
+fn test_longhands_parse_origin() {
+    let url = ServoUrl::parse("http://localhost").unwrap();
+    let context = ParserContext::new(Origin::Author, &url, Box::new(CSSErrorReporterTest));
+
+    let mut parser = Parser::new("1px 2px rubbish");
+    let parsed = longhands::parse_origin(&context, &mut parser);
+    assert_eq!(parsed.is_ok(), true);
```

Comment:
The check for exhaustion is below this line, but I'm actually curious why this test is here since it doesn't seem relevant to the issue that's being resolved.